requirements.txt

```diff
@@ -2,3 +2,4 @@ glfw>=1.4.0
 numpy>=1.11
 Cython>=0.25.2
 imageio>=2.1.2
+imagehash>=3.7
```

Comment:
I don't think we need to add image-hash here, since it's only needed for testing (and is included in `requirements-dev.txt`). I realize this isn't clear at all from README. Sorry about that -- maybe you can add note to relevant section?

(it is, however, noted in setup.py https://github.com/openai/mujoco-py/blob/master/setup.py#L46 )